packages/grafana-ui/src/components/BarGauge/BarGauge.test.tsx

```diff
@@ -11,6 +11,7 @@ import {
   getDisplayProcessor,
   createTheme,
 } from '@grafana/data';
+import { BarGaugeDisplayMode } from '@grafana/ui';
```

Comment:
So I don't think that the `BarGaugeDisplayMode` change is actually breaking. However the move to `@grafana/schema` changes the built output so that the enum is no longer placed directly in the built ouput but rather is exported directly from `@grafana/schema`.

I'm not entirely sure how to work around this though in the build.

Reply:
OK, pretty sure the breaking change detection is not working in this case so going to mark that as complete.